targets/syslog.go

```diff
@@ -1,3 +1,4 @@
+//go:build !windows && !nacl && !plan9
```

Comment:
Do we need both `//go:build` and `// +build`?

Reply:
Thanks @mgdelacroix.  The build tag was converted automatically by gofmt.  It leaves the old line for compatibility with Go 1.16 and below.

[From the docs](https://pkg.go.dev/cmd/go#hdr-Build_constraints):
```
Go versions 1.16 and earlier used a different syntax for build constraints, with a "// +build" prefix. The gofmt command will add an equivalent //go:build constraint when encountering the older syntax. 
```